sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -66,15 +65,19 @@ export interface LogsColumn {
 export type LogsColumnType = string;
 
 // @public
-export type LogsQueryBatchOptions = OperationOptions;
+export interface LogsQueryBatchOptions extends OperationOptions {
+    throwOnAnyFailure?: boolean;
```

Comment:
Since we return a promise that can be rejected, I think `rejectOnAnyFailure` might be more accurate.

But remind me...so if I set this to true, then if any of the queries in the batch fails, the whole thing fails? The scenario you talked to me about before was disabling throwing an error on partial failures, so I'm curious about this.

Reply:
Yes - this one will throw for both complete and partial failures. And the whole thing will fail in this case. But the users will get the data about all the responses. We wanted to follow similar approach across all JS sdks - https://github.com/Azure/azure-sdk-for-js/blob/8a7f4e25fbbc25ea3ac1244ff1cf83326472f770/sdk/search/search-documents/src/searchClient.ts#L493 Search-documents uses that flag - so thought of using the same approach.

Reply:
Ah gotcha, if search-documents already uses that flag, then disregard my suggestion. Rather keep it consistent with `throwOnAnyFailure` then.